src/cljs/rems/administration/create_organization.cljs

```diff
@@ -0,0 +1,268 @@
+(ns rems.administration.create-organization
+  (:require [clojure.string :as str]
+            [medley.core :refer [indexed map-vals remove-nth]]
+            [re-frame.core :as rf]
+            [rems.administration.administration :as administration]
+            [rems.administration.components :refer [localized-text-field text-field]]
+            [rems.administration.items :as items]
+            [rems.atoms :as atoms :refer [enrich-user document-title]]
+            [rems.common.util :refer [+email-regex+ conj-vec]]
+            [rems.collapsible :as collapsible]
+            [rems.dropdown :as dropdown]
+            [rems.fetcher :as fetcher]
+            [rems.fields :as fields]
+            [rems.flash-message :as flash-message]
+            [rems.spinner :as spinner]
+            [rems.text :refer [text text-format]]
+            [rems.util :refer [navigate! post! put! trim-when-string]]))
+
+(rf/reg-event-fx
+ ::enter-page
+ (fn [{:keys [db]} [_ organization-id]]
+   {:db (assoc db
+               ::organization-id organization-id
+               ::available-owners nil
+               ::editing? (some? organization-id)
+               ::form {:type :organization/default})
+    :dispatch-n [[::available-owners]
+                 (when organization-id [::organization])]}))
+
+(rf/reg-sub ::organization-id (fn [db _] (::organization-id db)))
+(rf/reg-sub ::editing? (fn [db _] (::editing? db)))
+
+;;; form state
+
+(rf/reg-sub ::form (fn [db _] (::form db)))
+(rf/reg-event-db ::set-form-field (fn [db [_ keys value]] (assoc-in db (concat [::form] keys) value)))
```

Comment:
I'd have written `(into [::form] keys)` but it doesn't matter

Reply:
I think I tend to use `concat` for these, somehow reads clearer to me. I think we use concat in the other places for this.